### PR TITLE
Update to seqerakit

### PR DESCRIPTION
**Pull request by ejseqera**
- Updates entities to reflect newer changes to the showcase
- Add YAMLs to support use of `seqerakit` to create entities
- Update README accordingly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,107 @@
-# `seqeralabs/showcase` Workspace on Nextflow Tower
+# `seqeralabs/showcase` Workspace on Seqera Platform
 
 ## Infrastructure as code
 
-Most [Nextflow Tower](https://cloud.tower.nf/) entities such as Pipelines, Compute Environments and Datasets can be exported in JSON format via the [Nextflow Tower CLI](https://github.com/seqeralabs/tower-cli#nextflow-tower-cli). This is very useful for creating infrastructure as code to store the exact configuration options used to create these entities, and to share and track changes over time.
+Infrastructure as Code (IaC) provides the ability to provision and manage infrastructure through configuration files. Having IaC enables automation in set up, cconsistency and standardization in infrastructure, and documentation of how resources are configured.
```
Comment (laramiellindsey) [inline README.md:5]:
```suggestion
Infrastructure as Code (IaC) provides the ability to provision and manage infrastructure through configuration files. Having IaC enables automation in set up, consistency and standardization in infrastructure, and documentation of how resources are configured.
```

File: README.md [hunk 2]
```diff
@@ -1,27 +1,107 @@
...
+conda install seqerakit
+```
+
+Or:
+
+```bash
+pip install seqerakit
+```
+
+Please refer to the package's [installation guide](docs/installation.md) for more details on the required dependencies.
```
Comment (laramiellindsey) [inline README.md:83]: This link it broken

File: README.md [hunk 3]
```diff
@@ -1,27 +1,107 @@
...
+
+#### Running `seqerakit`
+
+Similar to executing the scripts above to import and create Pipelines, Compute Environments, and Datasets that are defined in JSON onto Seqera Platform, the [YAML files](#using-seqerakit) provided in this repository can also be used create `seqeralabs/showcase` entities on the Platform. For example:
```
Comment (laramiellindsey) [inline README.md:87]:
```suggestion
Similar to executing the scripts above to import and create Pipelines, Compute Environments, and Datasets that are defined in JSON onto Seqera Platform, the [YAML files](#using-seqerakit) provided in this repository can also be used to create `seqeralabs/showcase` entities on the Platform. For example:
```

File: scripts/bash/tw_pipelines_export.sh [hunk 4]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 WORKSPACE='seqeralabs/showcase'
-OUTDIR=../pipelines
+OUTDIR=/Users/eshajoshi/Development/update_showcase/ej-showcase/pipelines
```
Comment (laramiellindsey) [inline scripts/bash/tw_pipelines_export.sh:4]: Will the OUTDIR need to be changed here from your username?

**Review by laramiellindsey — APPROVED** [review]
Minor edits. Apologies for the delay on the review!

**Comment by ejseqera** [comment]
Closing in favour of #4

**Pull request closed without merging**